<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-P0FATQV" failures="2" tests="2" name="Test(failed)" time="0.159" errors="0" timestamp="2024-08-25T11:36:00 IST">
  <testcase classname="failure.ClassB" name="testCase3" time="0.041">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at failure.ClassB.testCase3(ClassB.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testCase3 -->
  <testcase classname="failure.ClassA" name="testCase1" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at failure.ClassA.testCase1(ClassA.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testCase1 -->
  <testcase classname="failure.ClassB" name="testCase4" time="0.0">
    <ignored/>
  </testcase> <!-- testCase4 -->
  <testcase classname="failure.ClassA" name="testCase2" time="0.0">
    <ignored/>
  </testcase> <!-- testCase2 -->
</testsuite> <!-- Test(failed) -->
